docs/blog/posts/04-introducing-inf2-runtime.md

```diff
@@ -0,0 +1,261 @@
+---
+date: 2024-02-01
+tags:
+  - embeddings
+  - integrations
+  - inferentia
+categories:
+  - infra
+  - embeddings
+  - asic
+authors:
+ - spillai
+links:
+  - posts/04-introducing-inf2-runtime.md
+---
+
+# Introducing the NOS Inferentia2 Runtime
+
+<img src="/docs/blog/assets/nos-inf2.jpg" width="100%">
+
+We are excited to announce the availability of the [AWS Inferentia2](https://aws.amazon.com/en/ec2/instance-types/inf2/) runtime on [NOS](https://github.com/autonomi-ai/nos) - a.k.a. our **[`inf2`](../../concepts/runtime-environments.md#🏃‍♂️-supported-runtimes)** runtime. This runtime is designed to easily serve models on AWS Inferentia2, a high-performance, purpose-built chip for inference. In this blog post, we will introduce the AWS Inferentia2 runtime, and show you how to trivially deploy a model on the AWS Inferentia2 device using NOS. If you have followed our previous tutorial on [serving LLMs on a budget (on NVIDIA hardware)](./03-serving-llms-on-a-budget.md), you will be pleasantly surprised to see how easy it is to deploy a model on the AWS Inferentia2 device using the NOS **[`inf2`](../../concepts/runtime-environments.md#🏃‍♂️-supported-runtimes)** runtime we provide.
+
+## 🚀 What is AWS Inferentia2?
+
+[AWS Inferentia2](https://aws.amazon.com/en/ec2/instance-types/inf2/) (Inf2 for short) is the second-generation inference accelerator from AWS. Inf2 instances raise the performance of Inf1 (originally launched in 2019) by delivering 3x higher compute performance, 4x larger total accelerator memory, up to 4x higher throughput, and up to 10x lower latency. Inf2 instances are the first inference-optimized instances in Amazon EC2 to support scale-out distributed inference with ultra-high-speed connectivity between accelerators. 
+
+Relative to the [AWS G5 instances](https://aws.amazon.com/ec2/instance-types/g5/) ([NVIDIA A10G](https://www.nvidia.com/en-us/data-center/products/a10-gpu/)), Inf2 instances promise up to 50% better performance-per-watt. Inf2 instances are ideal for applications such as natural language processing, recommender systems, image classification and recognition, speech recognition, and language translation that can take advantage of scale-out distributed inference. 
```

Comment:
should we quote the inf2 numbers here against say an A100 for reference?

Reply:
Will do this once we have the profiling numbers to compare, I could only find this stat relative to G5.